src/crappy/actuator/phidgets_stepper4a.py

```diff
@@ -81,6 +81,7 @@ def __init__(self,
     self._remote = remote
     self._switch_ports = switch_ports
     self._switches = list()
+    self._check_switch = True
```

Comment:
Explain in a comment why this attribute cannot be set normally by the user. Otherwise, people reading this code in the future can think it's a mistake and try to "correct" it.